FrontEndReact/src/View/Admin/Add/AddUsers/AdminAddUser.js

```diff
@@ -62,9 +62,9 @@ class AdminAddUser extends Component {
                 message += "Invalid Role!";
             } else if (!this.props.chosenCourse["use_tas"] && document.getElementById("role").value==="TA/Instructor") {
                 message += "Invalid Role!";
-            } else if (validator.isEmpty(document.getElementById("lms_id").value)) {
+            } /*else if (validator.isEmpty(document.getElementById("lms_id").value)) {
```

Comment:
Since we no longer need the commented out code, go ahead and delete the commented out code!

Reply:
Thank you @jdkingsbury for removing the commented out code!